README.rst

```diff
@@ -186,19 +186,19 @@ before creating that mapping.
 **They are not where accounts or credentials are stored - these live on home
 servers. Identity Servers are just for mapping 3rd party IDs to matrix IDs.**
 
-This process is very security-sensitive, as there is obvious risk of spam if it
+This process is very security-sensitive, as there is an obvious risk of spam if it
 is too easy to sign up for Matrix accounts or harvest 3PID data. In the longer
-term, we hope to create a decentralised system to manage it (`matrix-doc #712
+the term, we hope to create a decentralised system to manage it (`matrix-doc #712
 <https://github.com/matrix-org/matrix-doc/issues/712>`_), but in the meantime,
 the role of managing trusted identity in the Matrix ecosystem is farmed out to
 a cluster of known trusted ecosystem partners, who run 'Matrix Identity
 Servers' such as `Sydent <https://github.com/matrix-org/sydent>`_, whose role
 is purely to authenticate and track 3PID logins and publish end-user public
 keys.
 
-You can host your own copy of Sydent, but this will prevent you reaching other
-users in the Matrix ecosystem via their email address, and prevent them finding
-you. We therefore recommend that you use one of the centralised identity servers
+You can host your copy of Sydent, but this will prevent you from reaching other
```

Comment:
I think the word `own` was helpful here.

```suggestion
You can host your own copy of Sydent, but this will prevent you from reaching other
```

Reply:
okk